Interfaces/API/Dirac.py

```diff
@@ -860,19 +863,23 @@ def runLocal(self, job):
 
     self.log.info('Attempting to submit job to local site: %s' % DIRAC.siteName())
 
-    # If not set differently in the CS use the root from the current DIRAC installation
-    siteRoot = gConfig.getValue('/LocalSite/Root', DIRAC.rootPath)
-
-    os.environ['DIRACROOT'] = siteRoot
-    self.log.verbose('DIRACROOT = %s' % (siteRoot))
-    os.environ['DIRACPYTHON'] = sys.executable
-    self.log.verbose('DIRACPYTHON = %s' % (sys.executable))
+    # DIRACROOT is used for finding dirac-jobexec (it is normally set by the JobWrapper)
+    # We don't use DIRAC.rootPath as we assume that a DIRAC installation is already done at this point
+    os.environ['DIRACROOT'] = os.environ['DIRAC']
+    self.log.verbose('DIRACROOT = %s' % (os.environ['DIRACROOT']))
 
     if 'Executable' in parameters:
       executable = os.path.expandvars(parameters['Executable'])
     else:
       return self._errorReport('Missing job "Executable"')
 
+    if '-o LogLevel' in arguments:
```

Comment:
Do I understand correctly that you are forcing the logLevel to debug ? Is that really ok ? 
Also, there could be several `-o` options no ?

Reply:
Yes, but this is purely for running jobs "locally", meaning for test (BTW, for some historical reasons these test functions are in the API, and that is wrong of course). 
I can of course code a better solution than the one above (which was a quick and dirty one for running my own tests...).